lib/beaker-puppet/install_utils/foss_utils.rb

```diff
@@ -362,7 +362,8 @@ def install_puppet_agent_on(hosts, opts = {})
           opts[:puppet_agent_version] ||= opts[:version] #backwards compatability with old parameter name
           opts[:puppet_collection] ||= puppet_collection_for(:puppet_agent, opts[:puppet_agent_version]) || 'pc1'
```

Comment:
Should `puppet_collection_for` return a downcased collection instead?

Reply:
fair question, but since there is some time pressure due to caused failures from the earlier change & what you're suggesting is functionally equivalent, I'm going to move forward with merging & releasing this change as-is.

Please propose a new PR with your suggested change if you'd like to pursue this further in a later release.